ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11vertex.cpp

```diff
@@ -165,6 +165,7 @@ inline ID3D11InputLayout* vertex_format_layout(const enigma::VertexFormat* verte
   for (size_t i = 0; i < vertexFormat->flags.size(); ++i) {
     const pair<int, int> flag = vertexFormat->flags[i];
     D3D11_INPUT_ELEMENT_DESC *vertexLayoutElement = &vertexLayoutElements[i];
+    *vertexLayoutElement = { };
```

Comment:
@JoshDreamland this isn't a big deal, but I decided that my own programming style should always be to default initialize structs. Do you agree this is generally a good idea? For example, you don't know if somebody is going to change the API and add new members which unitialized will cause issues.

Is this much of a performance concern? I want to know specifically in the case of where you plan to and do in fact initialize all of the current members of a struct anyway, should you still default initialize like this anyway?